nncf/quantization/quantize_model.py

```diff
@@ -258,6 +258,7 @@ def compress_weights(
     all_layers: Optional[bool] = None,
     dataset: Optional[Dataset] = None,
     sensitivity_metric: Optional[SensitivityMetric] = None,
+    awq: Optional[bool] = None,
```

Comment:
It looks like PyTorch backend does not support AWQ. I think, it will be good to raise an error for PyTorch model if `awq = True.`

Reply:
Done